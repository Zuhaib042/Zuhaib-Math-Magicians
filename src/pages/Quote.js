import React from 'react';
import Header from '../components/Header';

const Quote = () => (
  <div>
    <Header />
    <div className="text-white mt-24 mx-auto max-w-lg">
      <p className="text-2xl prose-blockquote">
        Mathematics are the result of mysterious powers which no one
        understands, and which the unconscious recognition of beauty must play
        an important part. Out of an infinity of designs a mathematician chooses
        one pattern for beautyâ€™s sake and pulls it down to earth.
      </p>
      <cite className="font-light">
        -- Marston Morse
        <br />
        <em>American mathematician</em>
        <br />
      </cite>
    </div>
  </div>
);

export default Quote;
