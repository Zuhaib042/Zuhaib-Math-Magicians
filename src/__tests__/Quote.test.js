import { render, screen } from '@testing-library/react';
import Quote from '../pages/Quote';

describe('Quote page', () => {
  it('should render properly', () => {
    render(<Quote />);

    const quotePage = screen.getByTestId('1');

    expect(quotePage).toMatchSnapshot();
  });
});
