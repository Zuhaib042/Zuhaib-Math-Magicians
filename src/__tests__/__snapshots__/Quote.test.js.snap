// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote page should render properly 1`] = `
<p
  class="text-2xl prose-blockquote"
  data-testid="1"
>
  Mathematics are the result of mysterious powers which no one understands, and which the unconscious recognition of beauty must play an important part. Out of an infinity of designs a mathematician chooses one pattern for beautyâ€™s sake and pulls it down to earth.
</p>
`;
